extensions/amp-base-carousel/1.0/scroller.js

```diff
@@ -143,6 +170,7 @@ function ScrollerWithRef({children, loop, restingIndex, setRestingIndex}, ref) {
       ref={containerRef}
       onScroll={handleScroll}
       class={`${classes.scrollContainer} ${classes.hideScrollbar} ${classes.horizontalScroll}`}
+      snap={String(snap)}
```

Comment:
I made a note on JSS, but IMHO we should just translate this into a class.

Reply:
Done!